FirebaseCombineSwift/Sources/Auth/User+Combine.swift

```diff
@@ -54,12 +58,16 @@
     ///
     /// The publisher will emit events on the **main** thread.
     ///
-    /// - Parameter credential: A user-supplied credential, which will be validated by the server. This can be
+    /// - Parameter credential: A user-supplied credential, which will be validated by the server.
+    /// This can be
     ///   a successful third-party identity provider sign-in, or an email address and password.
```

Comment:
```suggestion
    /// - Parameter credential: A user-supplied credential, which will be validated by the server.
    ///   This can be a successful third-party identity provider sign-in, or an email address and
    ///   password.
```